pages/dkp/konvoy/2.2/release-notes/2.2.1/index.md

```diff
@@ -0,0 +1,105 @@
+
+For this release, we are maintaining the documentation sets for individual platform components Konvoy and Kommander, while publishing some combined DKP documentation for processes, such as Upgrading DKP version.
+
+DKP 2.2.x supports Kubernetes versions between 1.21.0 and 1.22.x. Any cluster you want to attach using DKP 2.2.x must be running a Kubernetes version in this range.
+
+### Supported versions
+
+| Kubernetes Support | Version |
+| ------------------ | ------- |
+| **Minimum**  | 1.21.0  |
+| **Maximum**  | 1.22.x  |
+| **Default**  | 1.22.0  |
+
+## Fixes and Improvements
+
+### CRS deployments spamming service-account-tokens (COPS-7267)
+
+When running a Management Cluster with Attached Clusters, when all clusters used Metallb and deployed it using a ClusterResourceSet. On all clusters, the metal-lb namespace was being spammed with service-account-tokens of the form 'controller-token-xxxx' and 'speaker-token-xxxx'.
+
+### Certs showing as updated but not reloading in Kommander pods (COPS-7212)
+
+Under certain conditions, when certificate objects from a bundle are not expired, the pods in the cluster were not reloading with new certificates and might show the current certificates as expired.
+
+### kube-oidc-proxy error: certificate signed by unknown authority (COPS-7217)
+
+When adding a new Attached Cluster to the Management Cluster, using a custom domain and TLS certificate issued by Let's Encrypt, the kube-oidc-proxy helm chart in the Attached Cluster did not complete installation and the associated pod returned an error.
+
+## Component updates
+
+When upgrading to this release, the following services and service components are upgraded to the listed version:
+
+| Common Application Name | APP ID                                         | Version | Component Versions                                                                                                                                    |
+| ----------------------- | ---------------------------------------------- | ------- | ----------------------------------------------------------------------------------------------------------------------------------------------------- |
+| Cert Manager            | cert-manager                                   | 1.7.1   | - chart: 1.7.1 `<br>`- cert-manager: 1.7.1                                                                                                          |
+| Chartmuseum             | chartmuseum                                    | 3.6.2   | - chart: 3.6.2 `<br>`- chartmuseum: 3.6.2                                                                                                           |
+| Containerd              | containerd                                     | 1.4.13  |                                                                                                                                                       |
```

Comment:
I don't know if this is going to be bumped up in this release. cc @dkoshkin and @dlipovetsky

Reply:
This component did not get updated in 2.2.1, Please omit this. Containerd is still on  1.14.11

Reply:
We are not planning on including a change to the containerd version for the 2.2.1 release

Reply:
@Colin1841  while you were on vacation, we decided NOT to hold 2.2.1 for this. We will fix the issue with containerd in a future patch.